plugins/modules/bootc_manage.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+
+# Copyright (c) 2024, Ryan Cook <rcook@redhat.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt
+# or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: bootc_manage
+author:
+- Ryan Cook (@cooktheryan)
+short_description: Bootc Switch and Upgrade
+description:
+    - This is a bootc module that manages the switching and upgrading of bootc.
+options:
+    state:
+        description:
+            - 'Control to apply the latest image or switch the image.'
+            - 'This is a string value.'
+            - 'NOTE: This will not reboot the system.'
+            - 'Please use the reboot module to reboot the system.'
+        required: true
+        type: str
+        choices: ['switch', 'latest']
+    image:
+        description:
+            - 'The image to switch to.'
+            - 'This is required when state is ''switch''.'
```

Comment:
Please use the semantic markup https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation
```suggestion
            - 'This is required when O(state=switch)'.'
```